hs/src/Reach/EPP.hs

```diff
@@ -559,7 +560,10 @@ be_c = \case
     fg_saves this
     let cm = CT_From at1 this <$> mkfrom_info ce_readSave
     let lm = ET_FromConsensus at1 this <$> mkfrom_info ee_readSave <*> s'l
-    return $ (,) cm lm
+    do
```

Comment:
No need for this do... you're already in a do

Reply:
If one do is good, two must be better.